ui/src/commonMain/kotlin/com/materiiapps/gloom/ui/components/LargeSegmentedButton.kt

```diff
@@ -26,12 +27,15 @@ import androidx.compose.ui.unit.dp
 fun RowScope.LargeSegmentedButton(
     icon: Any,
     iconDescription: String? = null,
-    text: String
+    text: String,
+    onClick: () -> Unit = {},
```

Comment:
I don't think marking onClick as optional in a button is a good idea

Reply:
that'll only be temporary, as the other buttons don't yet have a function

Reply:
Then make those buttons have empty lambdas? Follow the same style Material3 Button does

Reply:
oh yeah i forgor

Reply:
> I don't think marking onClick as optional in a button is a good idea

Please clarify what you mean by "Optional". onClick is not optional and instead has an empty lambda as you suggested here:

> Then make those buttons have empty lambdas

Reply:
Xinto means don't give the parameter a default empty lambda and instead specify it where the button is actually used (ie. the details tab)

Reply:
https://github.com/MateriiApps/Gloom/blob/ee46cd2d76b0c302b3c829e32f11ebf0f9a48c95/ui/src/commonMain/kotlin/com/materiiapps/gloom/ui/components/settings/SettingsItem.kt#L26 There are other references of this kind of implementation